crdsonnet/render.libsonnet

```diff
@@ -133,7 +135,15 @@ local d = import 'github.com/jsonnet-libs/docsonnet/doc-util/main.libsonnet';
       local xofParts = self.xofParts(schema + { _parents: super._parents[1:] });
 
       local merge(parts) =
-        if std.isObject(parts)
+        if engineType == 'ast'
+        then
+          [
+            member
+            for part in parts
+            if jutils.type(part) == 'field' && jutils.isObject(part.expr)
+            for member in part.expr.members
+          ]
+        else if engineType == 'dynamic' && std.isObject(parts)
```

Comment:
What if `parts` isn't an object, is that an undefined scenario?

Reply:
AFAIK currently an impossible scenario, I can remove that and let it error out if it isn't an object, probably making it easier to debug too.